data.js

```diff
@@ -74,29 +73,53 @@ exports.data = [
             label: "wwf",
         },
         description: {
-            content: "The WWF is active since 1961 and therefore a well experienced player within the environmental engagement game. The biggest Pro on WWF is that it is well known and People trust the organization. They do not have a precise focus on the climate crises but there is also a focus on it. One of their goals for 2030 is to reduce greenhouse gas emissions by 40%.",
+            content: "The WWF is active since 1961 and therefore a well experienced player within the environmental engagement game. The biggest pro of WWF is that it is well known and people trust the organization. They do not have a precise focus on the climate crises but there is also a focus on it. One of their goals for 2030 is to reduce greenhouse gas emissions by 40%.",
             updatedBy: "Andre", updatedAt: "2019-08-18"
         },
         established: {
             content: "[1961](http://wwf.panda.org/knowledge_hub/history/)",
             updatedBy: "Andre", updatedAt: "2019-08-06"
         },
         kpis: {
-            content: "- Actions done / pa \n- Participants in actions\n- Members\n- Press Mentions",
+            content: "- Actions done / pa \n- Participants in actions\n- Members\n- Press mentions \n- advised organization \n- Visitors on Website",
             updatedBy: "Andre", updatedAt: "2019-08-06",
-
         },
+        actions: {
+            content: "These vary. With their expertise as a big environmental player they advise (countries, communities and private companies)[https://www.worldwildlife.org/initiatives/climate] in the topic."+
```

Comment:
`These vary. ` -> `The big variety of actions is [available online](TODO LINK).\n- They advise (countries, communities and private companies)[https://www.worldwildlife.org/initiatives/climate] on many environmental topics.`

Reply:
changed the wording